File: hw/viteri/u2/HW18ZooInPython/Zoo/model/Feline.py
from datetime import date
from model.Mammal import Mammal
from model.Cage import Cage

class Feline(Mammal):
    def __init__(self, id, description, born_on_date, gender, cage, number_of_bones, vertebrate):
        super().__init__(id, description, born_on_date, gender, cage, number_of_bones, vertebrate)

File: ws/viteri/u2/WS24Zo0/Zoo/src/main/java/model/Feline.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package model;

import java.util.Date;

/**
 *
 * @author Alexis Viteri DCO-ESPE
 */
public class Feline extends Mammal{
    
    public Feline(int id, String description, Date bornOnDate, char gender, Cage cage, int numberOfBones, boolean vertebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate);
    }
    
}
